src/search/FileFilters.js

```diff
@@ -490,12 +491,13 @@ define(function (require, exports, module) {
         _picker.on("listRendered", _handleListRendered);
         
         _picker.on("select", function (event, item, itemIndex) {
+            // Trigger a exclusionSetChange Event which will be handled by findBar.
+            exports.trigger("exclusionSetChange");
```

Comment:
Is there any existing API which could be used to listen to change event of view-state? I see one existing for default PreferencesManager values ( PreferencesManager.on(event, pref, handler) )